### PR TITLE
fix(suite): Fix close button in suite desktop promo

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve 

## Screenshots:

before
image

after
image

### CODE REVIEW

File: packages/suite/src/views/settings/SettingsGeneral/DesktopSuiteBanner.tsx [hunk 1]
```diff
@@ -23,36 +24,14 @@ const Container = styled(motion.div)`
     overflow: hidden;
 `;
 
-// eslint-disable-next-line local-rules/no-override-ds-component
-const CloseButton = styled(Icon)`
+const CloseButtonContainer = styled.div`
```
Comment (seibei-iguchi) [inline packages/suite/src/views/settings/SettingsGeneral/DesktopSuiteBanner.tsx]: Use Box:D

**Review by seibei-iguchi — CHANGES_REQUESTED** [review]

**Comment by seibei-iguchi** [comment]
You should refactor the rest of the component, what do you think?:D

**Comment by jvaclavik** [comment]
> You should refactor the rest of the component, what do you think?:D

Sorry, not this time 😅, we have to merge this soon and there are too many other issues

File: packages/suite/src/views/settings/SettingsGeneral/DesktopSuiteBanner.tsx (hunk 1 above)
Comment (jvaclavik) [reply packages/suite/src/views/settings/SettingsGeneral/DesktopSuiteBanner.tsx]: Done

**Review by seibei-iguchi — APPROVED** [review]

**Comment by bosomt** [comment]
**QA OK**

image

**Info:**
 - Suite version: web 25.1.0 (7748d59e23ecbafea9e6479d4551edcd6453e9f1)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: 
 - Transport: WebUsbTransport

**Pull request merged**